bindgen-tests/tests/headers/issue-2556/LayoutConstants.h

```diff
@@ -0,0 +1,7 @@
+#include "nsSize.h"
+
+namespace foo {
+
+static constexpr nsSize kFallbackIntrinsicSize(0, 0);
+
+}  // namespace mozilla
```

Comment:
Should this say `foo`?

Reply:
sure, no reason for the comment really actually, since it's a tiny file.